### PR TITLE
OCP4: Kickstart PCI-DSS profile

### PR DESCRIPTION
**Pull request by JAORMX**
This starts the PCI-DSS profile by adding a couple of rules with their
respective references.

Signed-off-by: Juan Antonio Osorio Robles <jaosorior@redhat.com>

### CODE REVIEW

File: products/ocp4/profiles/pci-dss.profile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        - jhrozek
+        - mrogers950
+
+reference: https://www.pcisecuritystandards.org/documents/PCI_DSS_v3-2-1.pdf
+
+title: 'PCI-DSS v3.2.1 Control Baseline for Red Hat OpenShift Container Platform 4'
+
+description: |-
+    Ensures PCI-DSS v3.2.1 security configuration settings are applied.
+
```
Comment (jhrozek) [inline products/ocp4/profiles/pci-dss.profile:15]: is `platform: ocp4` implied from the path?

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now 349b023547e877359698b475ae30402fc00ff6f2

File: products/ocp4/profiles/pci-dss.profile (hunk 1 above)
Comment (JAORMX) [reply products/ocp4/profiles/pci-dss.profile:17]: It's probably infered from product.yml. But I added it anyway now that you suggested it. Explicit is better than implicit.

**Review by jhrozek — APPROVED** [review]
/lgtm
I think the CI issues of the moderate profile are clearly unrelated. We should probably investigate if they persist, but in the meantime, feel free to merge this PR.

**Pull request merged**